challenges/classes.js

```diff
@@ -1,7 +1,63 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+/*function CuboidMaker(attr) {
+    this.length = attr.length;
+    this.width = attr.width;
+    this.height = attr.height;
+}
+CuboidMaker.prototype.volume = function () {
+    return this.length * this.width * this.height;
+};
+CuboidMaker.prototype.surfaceArea = function () {
+    return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
+};
+*/
+
+class CuboidMakerAgain {
+	constructor(attr) {
+		this.length = attr.length;
+		this.width = attr.width;
+		this.height = attr.height;
+	}
+	volume() {
+		return this.length * this.width * this.height;
+	}
+	surfaceArea() {
+		return 2 * (this.length * this.width + this.length * this.height + this.width * this.height);
```

Comment:
please use brackets to mark the separation  between 2 operator. so, it's easier to read do something like

`return 2 * ((this.length * this.width) + (this.length * this.height) + (this.width * this.height));`